website/i18n/en/docusaurus-plugin-content-docs/current/about/mission.md

```diff
@@ -4,47 +4,49 @@ sidebar_position: 1
 
 # Mission
 
-Здесь описаны цели и ограничения применимости методологии - которыми мы руководствуемся при разработке методологии
+Here we describe the goals and limitations of the applicability of the methodology-which we are guided by when developing the methodology
 
-- Мы видим нашу цель, как баланс между идеологией и простотой
-- Мы не сможем сделать серебряную пулю, которая подходит всем
+- We see our goal as a balance between ideology and simplicity
+- We won't be able to make a silver bullet that fits everyone
 
-**Тем не менее, хотелось бы, чтобы методология была близка и доступна достаточно обширному кругу разработчиков**
+**Nevertheless, I would like the methodology to be close and accessible to a fairly wide range of developers**
 
-## Цели
+## Goals
 
-### Интуитивная понятность для широкого круга разработчиков
+### Intuitive clarity for a wide range of developers
 
-Методология должна быть доступна - большей части команды в проектах
+The methodology should be available - for most of the team in projects
 
-*Т.к. даже со всем будущим инструментарием - будет недостаточно того, чтобы методологию понимали только проженные сеньоры/лиды*
+*Because even with all the future tools , it will not be enough for only experienced seniors/leads to understand the methodology*
```

Comment:
Текущая формулировка звучит странновато, но мб тут я и не прав и более прошаренные англичане меня поправят
```suggestion
*Because even with all the future tools , it will not be enough, if only experienced seniors/leads will understand the methodology*
```